recipes/dirent/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "dirent" %}
+{% set version = "1.21" %}
+{% set sha256 = "b5214abdd138056f893bad41e6f15081cb454304df9432c1d233e5a68eddf839" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://softagalleria.net/download/dirent/dirent-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script:
+    - if not exist "%LIBRARY_INC%" mkdir "%LIBRARY_INC%"  # [win]
+    - copy "include\\dirent.h" "%LIBRARY_INC%\\"          # [win]
+
+test:
+  commands:
+    - if not exist "%LIBRARY_INC%\\dirent.h" exit 1       # [win]
+
+about:
+  home: https://softagalleria.net/dirent.php
+  license: MIT
+  # No license file, unfortunately.
```

Comment:
License file is not included unfortunately. However the header says MIT. It looks like they are moving it to a [GitHub repo]( https://github.com/tronkko/dirent ). Though there is no release/tag and it is unclear how that relates to package on `softagalleria.net`. So for now went with the `softagalleria.net` package, but can switch over and package the license once those issues are addressed. Opened issue ( https://github.com/tronkko/dirent/issues/10 ) upstream to get the release situation addressed.